### PR TITLE
port to python 38

### PR DESCRIPTION
**Pull request by bissenbay**
Related-To: https://github.com/thoth-station/thoth-application/issues/545

### CODE REVIEW

File: .thoth.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ runtime_environments:
     operating_system:
       name: ubi
       version: "8"
-    python_version: "3.6"
+    python_version: "3.8"
```
Comment (bissenbay) [inline .thoth.yaml:10]: @fridex @harshad16 do I need to do this change in `.thoth.yaml` for all repositories as well?

**Comment by bissenbay** [comment]
@harshad16 when `aicoe-ci/build-check` fails, where I should look for error? I go to `Details`, but I don't see logs etc., how to trace down the error in such cases?

File: .thoth.yaml (hunk 1 above)
Comment (fridex) [reply .thoth.yaml:10]: Yes, please do so. :+1:

Comment (bissenbay) [reply .thoth.yaml:10]: thanks

**Comment by bissenbay** [comment]
@fridex @harshad16 Do we need to add `.aicoe-ci.yaml` in this repo?

**Comment by harshad16** [comment]
@bissenbay ,
can you please add the  following context to `.aicoe-ci.yaml`
```
check:
  - thoth-build
  - thoth-precommit
build:
  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:latest
  build-stratergy: Source
  registry: quay.io
  registry-org: thoth-station
  registry-project: advise-reporter
  registry-secret: thoth-station-thoth-pusher-secret
deploy:
  project-org: "thoth-station"
  project-name: "thoth-application"
  image-name: "advise-reporter"
  overlay-contextpath: "advise-reporter/overlays/test/imagestreamtag.yaml"
```

**Comment by harshad16** [comment]
> when `aicoe-ci/build-check` fails, where I should look for error? I go to `Details`, but I don't see logs etc., how to trace down the error in such cases?

once pipeline task are visible, then traverse to `pr-build-run` > `run-build` .
check the log, at this point, it should have all the relevant information.

**Comment by bissenbay** [comment]
> @bissenbay ,
> can you please add the following context to `.aicoe-ci.yaml`
> 
> ```
> check:
>   - thoth-build
>   - thoth-precommit
> build:
>   base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:latest
>   build-stratergy: Source
>   registry: quay.io
>   registry-org: thoth-station
>   registry-project: advise-reporter
>   registry-secret: thoth-station-thoth-pusher-secret
> deploy:
>   project-org: "thoth-station"
>   project-name: "thoth-application"
>   image-name: "advise-reporter"
>   overlay-contextpath: "advise-reporter/overlays/test/imagestreamtag.yaml"
> ```

thanks @harshad16 , https://github.com/thoth-station/advise-reporter/pull/143/commits/9d603fa184e4da0763a0c7f66f7d34eba3154bd5

**Review by harshad16 — APPROVED** [review]
lgtm, thanks for all the changes.

**Review by fridex — APPROVED** [review]
changes look good to me, thanks! :ok: :+1:

**Pull request merged**